### DISCUSSION TITLE
Meeting Agenda 2/6

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/senior-design#259] [Meeting Agendas/Minutes]
1. Review tasks from previous sprint
2. Sprint Planning

@dyc3:
1. Make sure everyone is watching both repos for notifs
2. Demo my grafana and how it works
3. Demo cool terminal productivity stuff
    - tab complete
    - command history
4. Go over business model canvas, maybe finish it

**Comment by moreno-michael** [comment]
Meeting minutes 2/6

**Sprint Task Review**
1. Issue #1262 is halting progress for other aspects of project. Moving forward, Carson and Chris will work together by pair programming to resolve issue.
2. New approach to Grafana data source approach, our own data sets will be developed independent of Grafana (now issue #1300)
3. Issue #1239 still in progress due to issues with dev environment that have been resolved. Issue is pushed to the next sprint
4. Issue #1251 completed

**Sprint Planning**
1. Issue #1296 and #1297 assigned to Victor
2. Issue #1295 assigned to Michael
3. Issue #1262 pushed to this sprint

-Grafana demo
-Starship.rs demo
-Business model canvas completed and submitted for IDE
